core/perf_test/test_atomic_minmax_simple.cpp

```diff
@@ -150,6 +144,60 @@ void test(const int length) {
     }
     std::cout << "Time for 100% min early exits: " << time << std::endl;
   }
+
+  // limit iterations for contentious test, takes ~50x longer for same length
+  auto con_length = length / 5;
+  // input is min values - some max atomics will replace
+  {
+    inp(0) = min;
```

Comment:
`inp` is a view on the device this line is an invalid memory access

Reply:
Aha... ok, what's the best way to initialize the value? Just a parallel_for with a range of 1?

Reply:
Yup.  Or if you just want to fill the whole view `deep_copy(inp, min);`

Reply:
Ok, now I'm somewhat more confused.  I see other places where `inp(0)` or similar are accessed for reads, but several of those predate my changes.  This is now failing in Jenkins, but running with `Kokkos_ENABLE{_DEBUG,_BOUNDS_CHECKING}` under ctest seems to pass locally.  Is there another good way to try and track down where this is going wrong?

Reply:
Think I found it, it was the check for success or error.  Still never actually got the error to pop on a local machine, so I must be missing an option, but hopefully this will solve it.